src/styles/header/header.scss

```diff
@@ -0,0 +1,63 @@
+.header {
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  display: flex;
```

Comment:
According to the code style rules, `display: flex;` should be the first property in the ruleset. Please, reorder your properties.